### PR TITLE
[project-base] fixed usage of sed command on macos

### PR DESCRIPTION
**Pull request by s3tezsky**
| Q             | A
| ------------- | ---
|Description, reason for the PR| On MacOS when running script `install.sh` there was created duplicity of `docker-compose.yml` named as `docker-compose.yml-E` because of wrong usage of `sed` command. This PR will fix it.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: project-base/.travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
             script:
                 - sudo apt install ruby
                 - gem install docker-sync
-                - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                - sed -i -r "s#sed -i \'\' -E#sed -i -r#" ./scripts/install.sh
```
Comment (s3tezsky) [inline project-base/.travis.yml]:
```suggestion
                - sed -i -r "s#sed -i '' -E#sed -i -r#" ./scripts/install.sh
```
Escaping is wrong here

File: upgrade/UPGRADE-v8.1.0-dev.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ There you can find links to upgrade notes for other versions too.
                     script:
                         - sudo apt install ruby
                         - gem install docker-sync
-                        - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                        - sed -i -r "s#sed -i \'\' -E#sed -i -r#" ./scripts/install.sh
```
Comment (grossmannmartin) [inline upgrade/UPGRADE-v8.1.0-dev.md]: Is it needed when it's running on Linux? And shouldn't it be without backslash as it's in .travis file?

File: project-base/scripts/install.sh [hunk 3]
```diff
@@ -55,8 +55,8 @@ case "$operatingSystem" in
         cp -f docker/conf/docker-compose-mac.yml.dist docker-compose.yml
         cp -f docker/conf/docker-sync.yml.dist docker-sync.yml
 
-        sed -i -E "s#www_data_uid: [0-9]+#www_data_uid: $(id -u)#" ./docker-compose.yml
-        sed -i -E "s#www_data_gid: [0-9]+#www_data_gid: $(id -g)#" ./docker-compose.yml
+        sed -i '' -E "s#www_data_uid: [0-9]+#www_data_uid: $(id -u)#" ./docker-compose.yml
+        sed -i '' -E "s#www_data_gid: [0-9]+#www_data_gid: $(id -g)#" ./docker-compose.yml
```
Comment (grossmannmartin) [inline project-base/scripts/install.sh]: 👍

File: project-base/.travis.yml [hunk 4]
```diff
@@ -19,7 +19,7 @@ jobs:
             script:
                 - sudo apt install ruby
                 - gem install docker-sync
-                - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                - sed -i -r "s#sed -i '' -E#sed -i -r#" ./scripts/install.sh
```
Comment (grossmannmartin) [inline project-base/.travis.yml]: Is it needed when it's running on Linux?

Comment (s3tezsky) [reply project-base/.travis.yml]: It is, we are simulating istaallation for macOS, where is used sed like `sed -i '' -E` and that needs to be replaced to linux version `sed -i -r`.

File: upgrade/UPGRADE-v8.1.0-dev.md (hunk 2 above)
Comment (s3tezsky) [reply upgrade/UPGRADE-v8.1.0-dev.md]: It is needed.
You are right about the escaping! Thanks.

File: project-base/.travis.yml (hunk 4 above)
Comment (grossmannmartin) [reply project-base/.travis.yml]: Oh, I see. I didn't realize that. Nice!

**Pull request merged**